chucks-pick3-web/lib/bootstrapFormValLogin.js

```diff
@@ -19,38 +33,101 @@
 
 }());
 
-// we created new function to group both checkFirstName and checkLastName then we changed error message
-// to be more general for both by pulling normal case method then changed hasData function to a general name check name for validity ,
-// then erase  checkFirstName and checkLastName function data and invoked checkedName function in both or just change oninput to checkName(this)
 
 
+<<<<<<< HEAD
```

Comment:
You may have a merge conflict here. You need to resolve this first by doing a rebase.